src/docs/get-started/web.md

```diff
@@ -59,6 +59,12 @@ $ flutter upgrade
 $ flutter config --enable-web
 ```
 
+{{site.alert.warning}}
```

Comment:
I thought that one of your primary concerns was that changing channels is slow because it doesn't retain both the original channel and the new channel? It's a full replace? (Also, indent the note by 2 spaces per line.)

How about:

```
{{site.alert.warning}}
  Running `flutter channel beta` replaces your current version of Flutter
  with the beta version and can take time if your connection is slow.
  After this, running `flutter upgrade` upgrades your install to the latest
 `beta`.  Returning to the stable channel (or any other) requires calling
 `flutter channel <channel>` explicitly.
{{site.alert.end}}
```

@jonahwilliams, can you vet this phrasing please?

Reply:
That seems reasonable to me (though it is more evidence that we need non-destructive channel updates...)

Reply:
YES YES YES! I have to often change channels and I try to avoid it.  :D

Reply:
@sfshaza2 @jonahwilliams I'm Making the changes right away